earth2mip/inference_medium_range.py

```diff
@@ -120,7 +120,7 @@ def run_forecast(
 
     nlat = len(model.grid.lat)
     channels = [
-        data_source.channel_names.index(name) for name in model.out_channel_names
+        data_source.channel_names.index(name) for name in model.in_channel_names
```

Comment:
Rather than switching to input channels, we should maybe instead only select the channels that are in the datasource (I'm not even sure why we do this line here...)

Honestly, maybe just out right eliminating likes 123-125 is the move. I'll check this today and circle back.